resources/public/api/conf/1.0/amend_employment_expenses.raml

```diff
@@ -0,0 +1,38 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.formatValue
```

Comment:
The format value documentation scenario doesn't match the spec, though may be some spec inconsistencies